Grbl_Esp32/src/Limits.cpp

```diff
@@ -305,33 +305,30 @@ void limits_go_home(uint8_t cycle_mask) {
     motors_set_homing_mode(cycle_mask, false);  // tell motors homing is done
 }
 
-uint8_t limit_pins[MAX_N_AXIS][2] = { { X_LIMIT_PIN, X2_LIMIT_PIN }, { Y_LIMIT_PIN, Y2_LIMIT_PIN }, { Z_LIMIT_PIN, Z2_LIMIT_PIN },
-                                      { A_LIMIT_PIN, A2_LIMIT_PIN }, { B_LIMIT_PIN, B2_LIMIT_PIN }, { C_LIMIT_PIN, C2_LIMIT_PIN } };
-
 uint8_t limit_mask = 0;
 
 void limits_init() {
-    limit_mask = 0;
-    int mode   = INPUT_PULLUP;
+    limit_mask     = 0;
+    Pin::Attr mode = Pin::Attr::Input | Pin::Attr::PullUp;
 #ifdef DISABLE_LIMIT_PIN_PULL_UP
```

Comment:
#define DISABLE_LIMIT_PIN_PULL_UP being omited from the machine file caused another recurring crash on reboot. Probably need a new way to make the decision about pullup.